content/docs/arrl/_index.md

```diff
@@ -13,7 +13,7 @@ icon: book
 icon_pack: fas
 ---
 
-![ARRL The National Association for Amateur Radio](logo.jpg)
+![ARRL The National Association for Amateur Radio](../images/logo.jpg)
```

Comment:
the correct link is images/logo.png et. al. for this top-level page.

Reply:
Is that a change? logo.jpg is what was there; I simply added the images directory. 

Logo also appears to be the only jpg in the directory.

Reply:
I "discovered" this error by running Hugo locally as described in [maintainer.md](https://github.com/HamStudy/examtools-docs/blob/master/maintainer.md#test-your-changes).

![image](https://user-images.githubusercontent.com/4356471/101992389-cdbb0b80-3c80-11eb-852b-ee7ed79edb59.png)

Referencing `../images` is correct in checkist.md and greeter-script.md but, I assume due to some Hugo weirdness, not for _index.md:

```
arrl$ tree
.
├── _index.md
├── checklist.md
├── greeter-script.md
└── images
    ├── expenses.png
    ├── invite.png
    ├── invoice.png
    ├── logo.jpg
    ├── schedule.png
    ├── test-report.png
    ├── upload-finder.png
    └── upload.png
```

Hugo turns `_index.md` into `http://localhost:1316/docs/arrl/` but `checklist.md` into a new subdirectory: `http://localhost:1316/docs/arrl/checklist/` so even though the files are at the same level in the source hierarchy, they are not when rendered. This explains the difference between needing `images/` in `_index.md` vs `../images/` in the other files.

You really need to render in Hugo locally to see all the weird crap that happens.

Reply:
That's weird. Rendering it locally is how I'm previewing, and it worked for me. 

Either way I'll get it changed.

Reply:
If you view this locally with a normal markdown viewer it looks fine; You have to view it with hugo to see the weird special handling it does.

Reply:
I'm rendering it with Hugo and using chrome.